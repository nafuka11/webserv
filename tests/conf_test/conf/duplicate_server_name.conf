server {
    server_name webserv01;
    server_name webserv02;
}