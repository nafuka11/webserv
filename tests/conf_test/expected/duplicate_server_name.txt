webserv: duplicate "server_name" directive in ./conf/duplicate_server_name.conf:3
